src/components/Article.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import CommentsList from './CommentsList'
+import Vote from './Vote';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState([]);
```

Comment:
as article will be an object, it'd be better if it's initialised as an empty object instead of an empty array. Just to keep the same datatype